pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "hatchling.build"
 
 [project]
 name = "fraim"
-version = "0.5.0"
+version = "0.5.1"
```

Comment:
We need a better release process, lol.  This should have been bumped to `0.5.1-dev` or similar (on `main`) immediately after `0.5.0` was tagged.

Reply:
yeah I also want to implement freezing deps in here. will make an issue for both of these